src/theme.js

```diff
@@ -5,6 +5,23 @@ const theme = {
         mediaThickS: 142,
         mediaSizeS: 470,
         mediaSizeL: 935
```

Comment:
Created breakpoints for tablet and mobilephone
       mediaThickS: 142,
        mediaSizeS: 470,
        mediaSizeL: 935
. Can merge it and later i gonna move mine to yours - breakpoints